modules/CuckooFilter/src/main.rs

```diff
@@ -1,3 +1,57 @@
-fn main() {
-    println!("Hello, world!");
-}
+extern crate redis;
+use redis::Commands;
+use probabilistic_collections::cuckoo::CuckooFilter;
+
+fn main() {
+    let size = std::env::args().nth(1).expect("Please specify the size of the filter");

```

Comment:
**[Desired, Maintainability]** Instead of repeating `std::env` put a `use` statement at the top of the file: `use std::env;`